code/datums/diseases/parrotpossession.dm

```diff
@@ -0,0 +1,27 @@
+/datum/disease/parrot_possession
+	name = "Parrot Possession"
+	max_stages = 1
+	spread_text = "Paranormal"
+	spread_flags = SPECIAL
+	disease_flags = CURABLE
+	cure_text = "Holy Water."
+	cures = list("holywater")
+	cure_chance = 20
+	agent = "Avian Vengence"
+	viable_mobtypes = list(/mob/living/carbon/human)
+	desc = "Subject is possesed by the vengeful spirit of a parrot. Call the priest."
+	severity = MEDIUM
+	var/mob/living/simple_animal/parrot/Poly/ghost/parrot
```

Comment:
yes, it's an easter egg where a poly that is repeatedly slaughtered comes back as an unkillable ghost.

You know, for fun.